src/GeoIP/GeoIP.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import MMDB
 
 open class GeoIP {
-    open static let database = MMDB()!
+    open static let database = MMDB()
```

Comment:
Another merge request worth doing will be to remove GeoIP completely and remove the MMDB dependancy